packages/react/src/components/InAppMessaging/MessageLayout/__tests__/utils.spec.ts

```diff
@@ -20,10 +21,14 @@ describe('getButtonModifier', () => {
     let expected: MessageLayoutButtonModifier = 'light';
     // light by default
     expect(getButtonModifier({})).toBe(expected);
-    expect(getButtonModifier(null)).toBe(expected);
+    expect(getButtonModifier(null as unknown as React.CSSProperties)).toBe(
```

Comment:
Does `unknown` gobble up `React.CSSProperties`?

Reply:
Casting to `unknown` is to allow the `null` usage when calling `getButtonModifier` (not sure if that answers your question 🦃 )